radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_MENU_LOGICAL_SWITCHES       STR_CHAR_SWITCH "Логічні перемикачі"
+#define TR_MENU_TRAINER                STR_CHAR_TRAINER "Трейнер"
+#define TR_MENU_CHANNELS               STR_CHAR_CHANNEL "Канали"
+#define TR_MENU_GVARS                  STR_CHAR_SLIDER "ГлобЗмін"
+#define TR_MENU_TELEMETRY              STR_CHAR_TELEMETRY "Телеметрія"
+#define TR_MENU_DISPLAY                "ДИСПЛЕЙ"
+#define TR_MENU_OTHER                  "Інше"
+#define TR_MENU_INVERT                 "Інверт"
+#define TR_AUDIO_MUTE                  TR("Аудіо стоп","Тихо якщо нема звуку")
+#define TR_JITTER_FILTER               "фільтер ADC"
+#define TR_DEAD_ZONE                   "Мертва зона"
+#define TR_RTC_CHECK                   TR("Перевір RTC", "Перевір RTC напругу")
+#define TR_AUTH_FAILURE                "Помилка авторизації"
+#define TR_RACING_MODE                 "Режим гонки"
+
+#define TR_USE_THEME_COLOR              "Користуй колір теми"
+
+#define TR_ADD_ALL_TRIMS_TO_SUBTRIMS    "Довдати все трими до субтримів"
+#if LCD_W > LCD_H
+  #define TR_OPEN_CHANNEL_MONITORS        "Відкрити монітор каналів"
+#else
+  #define TR_OPEN_CHANNEL_MONITORS        "Відкр. мон.кан."
+#endif
+#define TR_DUPLICATE                    "Дублювати"
+#define TR_ACTIVATE                     "Встановити активним"
+#define TR_RED                          "Червоний"
+#define TR_BLUE                         "Блакитний"
+#define TR_GREEN                        "Зелений"
+#define TR_COLOR_PICKER                 "Обирач кольору"
+#define TR_EDIT_THEME_DETAILS           "Редагувати деталі теми"
+#define TR_THEME_COLOR_DEFAULT         "ЗАМОВЧ"
+#define TR_THEME_COLOR_PRIMARY1        "ОСНОВНИЙ1"
+#define TR_THEME_COLOR_PRIMARY2        "ОСНОВНИЙ2"
+#define TR_THEME_COLOR_PRIMARY3        "ОСНОВНИЙ3"
+#define TR_THEME_COLOR_SECONDARY1      "ВТОРИННИЙ1"
+#define TR_THEME_COLOR_SECONDARY2      "ВТОРИННИЙ2"
+#define TR_THEME_COLOR_SECONDARY3      "ВТОРИННИЙ3"
+#define TR_THEME_COLOR_FOCUS           "ФОКУС"
+#define TR_THEME_COLOR_EDIT            "РЕДАГУВАТИ"
+#define TR_THEME_COLOR_ACTIVE          "АКТИВНИЙ"
+#define TR_THEME_COLOR_WARNING         "WARNING"
+#define TR_THEME_COLOR_DISABLED        "ДЕАКТИВОВАНИЙ"
+#define TR_THEME_COLOR_CUSTOM          "КОРИСТУВАЦЬКИЙ"
+#define TR_THEME_CHECKBOX              "Checkbox"
+#define TR_THEME_ACTIVE                "Акстивний"
+#define TR_THEME_REGULAR               "Регулярний"
+#define TR_THEME_WARNING               "Warning"
+#define TR_THEME_DISABLED              "Деактивований"
+#define TR_THEME_EDIT                  "Редагувати"
+#define TR_THEME_FOCUS                 "Фокус"
+#define TR_AUTHOR                       "Автор"
+#define TR_DESCRIPTION                  "Опис"
+#define TR_SAVE                         "Зберігти"
+#define TR_CANCEL                       "Відміна"
+#define TR_EDIT_THEME                   "РЕДАГУВАТИ ТЕМУ"
+#define TR_DETAILS                      "Деталі"
+#define TR_THEME_EDITOR                 "ТЕМИ"
```

Comment:
Redundant space here
```suggestion
#define TR_AUTHOR                      "Автор"
#define TR_DESCRIPTION                 "Опис"
#define TR_SAVE                        "Зберігти"
#define TR_CANCEL                      "Відміна"
#define TR_EDIT_THEME                  "РЕДАГУВАТИ ТЕМУ"
#define TR_DETAILS                     "Деталі"
#define TR_THEME_EDITOR                "ТЕМИ"
```